plugins/domains/scribd.com/scribd.com-error.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+
+    re: [
+        /^https?:\/\/(?:www|\w{2})\.scribd\.com\/(doc|document|embeds|presentation|fullscreen)\/(\d+)/i
```

Comment:
We can actually import regexp from the main plugin file via require as  `re: require(...).re` - please take a look at other plugins (say, facebook fallbacks).